wbs/1.02C.09.tex

```diff
@@ -0,0 +1,16 @@
+\subsection{1.02C.09 Integration and Test}
+
+This WBS element covers integration and test activities. It includes:
+
+\begin{itemize}
+
+\item{Test and QA Data Curation: Curation of fixed data sets and
+  recommended tests assembled to provide a small but rich set of test
+  data for 10.2.1.01}
+  
+\item{Development of testing specifications and operation rehearsals
```

Comment:
This seems to me to be a rather limited definition of "integration".  The things mentioned here are clearly important, but there seems to be something perhaps even larger that's missing.

From the experience of putting together PDAC version 1 last fall, there was a lot of specific coordination work required to pull together all the pieces and ensure that they were deployed in a coordinated fashion.  PDACv1 was still a pretty simple system and that work could be handled as LOE work by a single person, but for the full integration of DM into functioning systems, there will very likely need to be properly scheduled effort on the part of multiple people in a central team.  Is that in some other WBS?  It looks like it might be in 02C.07.01 by its current definition.

If that's right, and "Integration" in that sense is not part of 02C.10, then maybe something like "System-Level Testing and Verification" might be a better name for 02C.10?